app/views/landing/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<%= render partial: 'header' %>
+<div class="shift-down">
+	<div class="row medium"></div>
+	<div class= "row-break"></div>
+	
+	<% @products.each_with_index do |products, index| %>
+	<div class= "row container-trim-wide"><h5 class="cat-heading"><%= @categories[index].upcase %></h5></div>
+	<div class="container-large">
+		<div class= "row">
+			<%= render partial: "product", collection: products %>
```

Comment:
I don't really get the "Lol" here. However, I'm not putting *every product* in one row. I'm putting *each category* in one row. All four products under that category are rendered in that row, while on a large screen.